### PR TITLE
Move the feed reject helper to document module.

### PR DESCRIPTION
**Pull request by baldersheim**
@geirst PR
@vekterli or @toregge FYI

### CODE REVIEW

File: document/src/tests/documenttestcase.cpp [hunk 1]
```diff
@@ -1251,4 +1265,42 @@ TEST(DocumentTest, testDeserializeMultiple)
     EXPECT_EQ(correct, sv3);
 }
 
+TEST(DocumentRejectTest, requireThatFixedSizeFieldValuesAreDetected) {
```
Comment (geirst) [inline document/src/tests/documenttestcase.cpp:1268]: Consider putting these tests in a separate test file. documenttestcase.cpp is already pretty bloated.

**Review by geirst — APPROVED** [review]
👍

File: document/src/tests/documenttestcase.cpp (hunk 1 above)
Comment (baldersheim) [reply document/src/tests/documenttestcase.cpp]: Fixed

File: document/src/tests/feed_reject_helper_test.cpp [hunk 2]
```diff
@@ -0,0 +1,74 @@
+// Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```
Comment (geirst) [inline document/src/tests/feed_reject_helper_test.cpp:1]: Please update copyright.

Comment (baldersheim) [reply document/src/tests/feed_reject_helper_test.cpp:1]: Will be done in upcomming PR

**Pull request merged**